### PR TITLE
Test is being flaky

### PR DESCRIPTION
**Pull request by littleaj**
I think this is why the test is failing

### CODE REVIEW

File: web/src/test/java/com/microsoft/applicationinsights/web/internal/httputils/HttpServerHandlerTest.java [hunk 1]
```diff
@@ -161,7 +157,11 @@ public void timeStatusCodeAreSetWhenHandleEndIsInvoked() throws MalformedURLExce
         assertThat(rt.getUrl().toString(), equalTo(url));
         assertThat(rt.getContext().getUser().getUserAgent(), equalTo("User-Agent"));
         assertThat(rt.getTimestamp(), is(CoreMatchers.<Date>notNullValue()));
-
+        try {
+            TimeUnit.MILLISECONDS.sleep(100); // pause for a moment
```
Comment (trask) [inline web/src/test/java/com/microsoft/applicationinsights/web/internal/httputils/HttpServerHandlerTest.java]: Can you add comment why `sleep` is needed?

File: web/src/test/java/com/microsoft/applicationinsights/web/internal/httputils/HttpServerHandlerTest.java [hunk 2]
```diff
@@ -161,7 +157,11 @@ public void timeStatusCodeAreSetWhenHandleEndIsInvoked() throws MalformedURLExce
...
+        } catch (InterruptedException e) {
```
Comment (trask) [inline web/src/test/java/com/microsoft/applicationinsights/web/internal/httputils/HttpServerHandlerTest.java]: No need to catch this

Comment (littleaj) [reply web/src/test/java/com/microsoft/applicationinsights/web/internal/httputils/HttpServerHandlerTest.java]: yup. will rethrow

File: web/src/test/java/com/microsoft/applicationinsights/web/internal/httputils/HttpServerHandlerTest.java (hunk 1 above)
Comment (littleaj) [reply web/src/test/java/com/microsoft/applicationinsights/web/internal/httputils/HttpServerHandlerTest.java]: will do

**Pull request merged**